ide/lsp.client/src/org/netbeans/modules/lsp/client/bindings/BreadcrumbsImpl.java

```diff
@@ -332,9 +330,12 @@ public LSPBreadcrumbPanel(SideBarFactory delegate, JTextComponent component) {
         }
 
         private void update() {
-            WORKER.post(() -> {
-                FileObject file = NbEditorUtilities.getFileObject(component.getDocument());
-                LSPBindings bindings = file != null ? LSPBindings.getBindings(file) : null;
+            FileObject file = NbEditorUtilities.getFileObject(component.getDocument());
+            final LSPBindings bindings = file != null ? LSPBindings.getBindings(file) : null;
+            if (bindings == null) {
+                return;
+            }
```

Comment:
This looks stange. I stumbled over the fact, that `getInitResult` can't block. It just returns the result of the initialization call to the LSP server, but `getBindings` might block as it might need to spin up a server and `update()` might be called from the EDT (it is called from the constructor. So from my POV this needs to be handled by the `IDLE_WORKER` (maybe `COMMON_WORKER` is a better name?